app/assets/images/payment_icons/ing_homepay.svg

```diff
@@ -0,0 +1 @@
368-.074-.515-.076 0-.154 0-.23-.074-.23 0-.539-.075-.539-.075 0 .223.079.59.079.81 0 .074.306.738.383.811a.577.577 0 00.307.294v.001zm7.294 4.889c-.076-.228-.304-.305-.458-.456 0 0-.994-.38-1.149-.38-.153.075.384.38.46.531.306.305 1.071.988 1.071.988l.076-.683zm-9.372-9.301c-.076.073-.154.221-.154.294-.154.221-.381.369-.23.59.076.295.076.589.308.81.23.222.534.147.84.368 0 0-.23-.958-.38-1.4-.077-.293-.23-.44-.384-.662zm12.236 10.837c-.304-.297-.38-.37-.456-.668.271-.085.551-.134.835-.148.379 0 .608-.073 1.062.076.089.043.312.217.527.352a2.003 2.003 0 01-.672.667c-.501-.125-.933-.066-1.296-.279zm-9.183-8.337c.154.592.383 1.332.46 2.217 0 .223-.077.444 0 .666l1.065.369s-.15-.887-.303-1.92c-.153-.886-.23-1.182-.458-1.553-.153.074-.229.074-.381.074-.131.041-.259.09-.383.147zm2.505 4.314c.153.147.228.44.304.953.076.294.23.367.38.806.23-.145.61-.072.61-.072 0-.367-.23-.734-.23-.806l-.38-.734c.076 0 .303.293.685.293.078 0 .306.072.457 0 0 0 .227.147.227.293.152.22.152.366.231.587.149 0 .227 0 .303-.073.304 0 .456-.074.456-.074-.075-.147-.152-.292-.152-.44-.077-.146-.227-.513-.227-.513l1.368-.22c0-.222-.075-.515-.305-.662-.684.222-1.443.294-1.977.294-.763 0-1.522-.147-2.587-.586-.076.146 0 .439 0 .439.456.369.456.147.837.515zM12.265 7.18c.535-.301 1.446-.828 1.75-1.128.078.15.991.677 1.37.9.304.228.61.302.913.451.152-.223.303-.45.457-.975.228.15.532.15.76.224 0 .3 0 .602.078.904 0 .148.151.073.456-.076.456-.15.912-.226 1.141-.3.53-.301.835-.902 1.292-.902.381 0 1.065.225 1.218.374.228.077.38.15.61.15.303 0 .531-.3.758-.45-.683-.45-1.442-.9-2.205-1.201-.457.374-.761.675-.989.675-.23 0-.076-.45-.076-.751-.229 0-.381 0-.608-.075-.15.525-.23 1.203-.609 1.352-.151.076-.23.076-.304.076v-.602c0-.075.304-.6.228-.675 0 0-.685.15-.836.225-.076.075-.535.301-.61.301 0 0-.229-.076-.304-.076-.228-.15-.38-.15-.609-.3-.303.902-.381 1.127-.381.977 0 0-1.066-.677-1.444-1.127-.99.15-.99.3-1.827.6-.304.3-.76.677-1.065 1.128-.152.3-.305.45-.456.75l.76.301c.077-.076.457-.6.532-.75zm13.007 5.02c-.077 0-.308-.073-.62-.146h-.386c.077.22.31.438.386.583 0 .074 0 .293.08.437.153 0 .385.073.54.146.077 0 .54.074.695.22.155 0 .694.582.774.655.23.146.463.292.616.219.155 0 0-.22 0-.292-.308-.437-.54-1.239-.616-1.239-.232-.072-.388 0-.696-.146-.154-.145-.618-.364-.773-.437zm-1.292-1.747c.076.151.224.151.374.226.15.15.3.226.524.377.298.302.447.68.82.979.15.076.225.152.299.152.076 0 .374.226.448.301 0-.603-.224-1.054-.372-1.357 0-.075-.076-.15-.225-.226-.226-.226-.449-.452-.523-.452-.223-.226-.373-.451-.597-.678-.225 0-.374.227-.524.302-.074.15-.371.302-.224.376zm-.276-1.92c0 .22.076.441.152.66.263.147.539.27.826.365.074-.22-.078-.439-.078-.66-.147-.512-.375-1.024-.45-1.61v-.294c-.298-.145-.602-.22-.977-.366 0 0 0 .44.15.66.152.366.377.952.377 1.245zm13.375 1.292l-.447.276-.598-.414.671-.346.745.484h-.371z" fill="#FF5300"/><path d="M37.06.94A1.5 1.5 0 0137.5 2v20a1.5 1.5 0 01-1.5 1.5H2A1.5 1.5 0 01.5 22V2A1.5 1.5 0 012 .5h34a1.5 1.5 0 011.06.44z" stroke="#000" stroke-opacity=".07"/></svg>
```

Comment:
Is it possible to move the icon outside of the border, not inside?

[![alt](https://screenshot.click/27-52-vhbi6-dj768.png)](https://screenshot.click/27-52-vhbi6-dj768.png)

Reply:
@tauthomas01 Anything is possible, but it's more in line with our brand if we keep it inside.

Reply:
@georgemaine Thank you for explaining. If that's the case, it's no problem.